### PR TITLE
Added 128-bits integer types support

### PR DESCRIPTION
**Pull request by Quentincestino**
xdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxdxd

### CODE REVIEW

**Comment by Supercip971** [comment]
awesome in my opinion

**Comment by nukeop** [comment]
Thanks, this is incredibly important for the future of this library.

**Comment by Quentincestino** [comment]
Yeah I know, it is pretty important to merge it as soon as possible in my opinion...

File: src/lib.rs [hunk 1]
```diff
@@ -14,4 +14,4 @@ macro_rules! prim_impl {
     };
 }
 
-prim_impl!(i8 u8 i16 u16 i32 u32 i64 u64);
+prim_impl!(i8 u8 i16 u16 i32 u32 i64 u64 i128 u128);
```
Comment (SuperFola) [inline src/lib.rs:17]: You should add tests as well for those, who knows, maybe the implementation will fail

Comment (Supercip971) [reply src/lib.rs:17]: Yeah that's really important !

**Review by SuperFola — APPROVED** [review]

**Review by StackDoubleFlow — APPROVED** [review]

**Review by Lauriethefish — APPROVED** [review]

**Comment by Fernthedev** [comment]
when the imposter is sus, check if it's odd

**Pull request merged**